app.js

```diff
@@ -1,47 +1,147 @@
-  secondsElement.innerHTML = pad(totalseconds) - (minutesElement.innerHTML*60) - (hoursElement.innerHTML*3600);
-  minutesElement.innerHTML = pad(parseInt(totalseconds/60)) - (hoursElement.innerHTML*60);
-  hoursElement.innerHTML = pad(parseInt(totalseconds/3600));      
+function createTarget() {
+  const boardParams = boardElement.getBoundingClientRect();
+  let x=Math.random()*(boardParams.width - 50);
+  x=Math.round(x);
+  let y=Math.random()*(boardParams.height - 50);
+  y=Math.round(y);
+  let paragrath = document.createElement('p');
+  paragrath.classList.add('target');
+  paragrath.style.left = x + 'px';
+  paragrath.style.top = y + 'px';
+  paragrath.addEventListener('mouseover', onHoveringTarget);
+  paragrath.addEventListener('click', onClickTarget);
+  boardElement.append(paragrath);
 }
 
-function startTheGame(){
+function deleteTarget(){
+  const targetElement = document.getElementsByClassName('target');
+  targetElement[0].remove();
+}
+
+function setTime() {
+  let target = document.getElementsByClassName('target');
+  if (totalmiliseconds == 0 && life > 0) {
+    lifeReduction();
+    deleteTarget();
+    if (target[0]){
+      deleteTarget();
+    }
+    createTarget();
+    createTarget();
+    reset();
+    startTimer();
+  }  else if (totalmiliseconds == 0 && life == 0){
+    lifeReduction();
+  }
+  --totalmiliseconds;
+  miliseconds.innerHTML = pad(totalmiliseconds) - (secondsElement.innerHTML*100)
+  secondsElement.innerHTML = pad(parseInt(totalmiliseconds/100));
+}
+
+function startTimer() {
+  isTimerRunning = true;
+  startAndResetButtonElement.textContent = 'stop';
+  intervalId = setInterval(setTime, 10);
+  
+}
+
+function reset () {
+  clearInterval(intervalId)
+  if (scoreBoardElement.innerHTML == 0) {
+    totalmiliseconds = 300
+  } else {
+    totalmiliseconds = Math.round(300 * (0.95 / scoreBoardElement.innerHTML));
+  }
+  miliseconds.innerHTML = 0
+  secondsElement.innerHTML = 0;
+}
+
+function stopTimer() {
+  isTimerRunning = false;
+  reset();
+  startAndResetButtonElement.textContent = 'Start the game';
+}
+
+function onClickstartTheGame() {
   if (isTimerRunning === false) {
-    isTimerRunning = true;
-    startAndResetButtonElement.textContent = 'stop';
-    intervalId = setInterval(setTime, 1000);
-    let paragrath = document.createElement('p');
-    boardElement.append("", paragrath);
-    paragrath.classList.add('board-element');
+    createTarget();
+    createTarget();
+    startTimer();
   } else if (isTimerRunning === true) {
-    isTimerRunning = false;
-    clearInterval(intervalId)
-    totalseconds = 0;
-    secondsElement.innerHTML = 0;
-    minutesElement.innerHTML = 0;
-    hoursElement.innerHTML = 0;
-    startAndResetButtonElement.textContent = 'Start the game';
+    stopTimer();
+    deleteTarget();
+    deleteTarget()
+    life = 3;
+    lifeBoardElement.innerHTML = 3;
+  }
+}
+
+function onHoveringTarget() {
+  console.log('hovering !');  
+}
+
+function onClickTarget() {
```

Comment:
Incorrect ! you always delete first target in the array ! and it should be removed the clicked target not the first one on the list !

It may happen you click one target while the game will remove the other.

Hint: use `event` argument